config/config_loader.go

```diff
@@ -325,6 +326,12 @@ func GetBaseConfig() *BaseConfig {
 	return baseConfig
 }
 
+func GetSslEnabled() bool {
+	return sslEnabled
+}
+func SetSslEnabled(enabled bool) {
```

Comment:
Is it necessary for change this flag？

Reply:
yes,this flag will add to url，and use in client